README.md

```
@@ -94,26 +96,70 @@ gh workflow run pr-batch-merge.yml -f batch_label="batch:jules"
 
 ## Our Mission
 
-At **ivi374forivi**, we are committed to:
+#### Discussion Templates (`.github/DISCUSSION_TEMPLATE/`)
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)